app/js/views/tag-item.jsx

```diff
@@ -13,28 +13,7 @@ export default class TagItem extends React.Component {
   }
 
   handleSetTag(evt) {
-    const checked = evt.target.checked;
```

Comment:
Yes. IIRC you were throwing in earlier version of this PR. Maybe a `console.error` would be good to have here too.